src/main/java/com/hpe/application/automation/tools/common/CompatibilityRebrander.java

```diff
@@ -16,14 +24,34 @@
  * Important note to mention is this class will only work after:
  * <ul>
  * <li> Package path is renamed from the old brand to the new one
- * <li> Only the various Descriptors (e.g. BuildStepDescriptor) use this function
  * </ul>
  */
 public class CompatibilityRebrander {
-    private final static String COM_HPE = "com.hpe";
-    private final static String COM_HP = "com.hp";
-    private final static String COM_MICROFOCUS = "com.microfocus";
-    private final static Logger LOG = Logger.getLogger(CompatibilityRebrander.class.getName());
+    private static final String COM_HPE = "com.hpe";
+    private static final String COM_HP = "com.hp";
+    private static final String COM_MICROFOCUS = "com.microfocus";
+    private static final String PACKAGE_NAME = "com.microfocus.application.automation.tools";
+    private static final Logger LOG = Logger.getLogger(CompatibilityRebrander.class.getName());
+
+    private CompatibilityRebrander() {}
+
+    /**
+     * This function hooks to the Jenkins milestone when the plugins are prepared to load
+     * And adds aliases for all package found classes
+     */
+    @Initializer(before = InitMilestone.PLUGINS_PREPARED)
+    public static void addAliasesToAllClasses() {
+        LOG.info("Adding alias for in old package names to add backward compatibility");
+        try {
+            Class[] classes = getClasses(PACKAGE_NAME);
+            for (Class c: classes) {
+                addAliases(c);
+            }
+        }
+        catch(ClassNotFoundException | IOException e) {
+            LOG.warning(e.getMessage());
```

Comment:
Better use error.
Additional in which cases is the exception is fatal? is there any such case?